Ds3/Runtime/Network.cs

```diff
@@ -67,18 +69,22 @@ public IWebResponse Invoke(Ds3Request request)
             using (var content = request.GetContentStream())
             {
                 do
-                {
+                {
+                    if (sdkNetworkSwitch.TraceInfo) { Trace.WriteLine("Request: " + request.GetType().ToString()); }

```

Comment:
It would be helpful if there was a message after the request returned that told us how long it took, and what the http status code of the request was.